front/client/components/Article.vue

```diff
@@ -8,50 +8,20 @@
           <p class="article-top-group">{{value.group}}</p>
         </div>
       </div>
-      <div class="article-description" v-html="getCategoryDescription"></div>
-      <div v-for="(value, index) in getCategoryItems" :key="index"
-        class="article-item">
-        <div class="article-header">
-          <p class="article-hd-textlg">{{ value.kind.contents }}</p>
-          <a-link openInNewTab :url="`http://hackage.haskell.org/package/${value.kind.contents}`" class="article-header-link">
-            (Hackage)
-          </a-link>
-          <p class="article-hd-textsm">{{ value.group }}</p>
-          <div class="article-header-icons">
-            <i class="fas fa-arrow-up"></i>
-            <i class="fas fa-arrow-down"></i>
-            <div class="header-func-icons">
-              <i class="fas fa-cogs"></i>
-              <i class="fas fa-times"></i>
-            </div>
-          </div>
-        </div>
-        <div class="article-content">
-          <p class="article-section-title">Summary</p>
-          <div class="article-description" v-html="value.description.html"></div>
-          <div class="flex-wrapper article-section pros-cons-box">
-            <div class="width-50">
-              <p class="article-section-title">Pros</p>
-              <ul v-for="(value, index) in value.pros" :key="index">
-                <li v-html="value.content.html"></li>
-              </ul>
-            </div>
-            <div class="width-50">
-              <p class="article-section-title">Cons</p>
-              <ul v-for="(value, index) in value.cons" :key="index">
-                <li v-html="value.content.html"></li>
-              </ul>
-            </div>
-          </div>
-          <div class="article-section">
-            <p class="article-section-title">Ecosystem</p>
-            <div v-html="value.ecosystem.html"></div>
-          </div>
-          <div class="article-section notes-box">
-            <p class="article-section-title">Notes</p>
-            <div v-html="value.notes.html"></div>
-          </div>
-        </div>
+      <div v-if="categoryDescription !== ''" class="article-description">
+        <div v-html="categoryDescription"></div>
+      </div>
+      <div v-for="(value, index) in getCategoryItems" :key="index">
```

Comment:
rename getCategoryItems to categoryItems. No need to name computed properties with "get" prefix